interop-scoring/main.js

```diff
@@ -0,0 +1,506 @@
+/* eslint-disable max-len */
+
+'use strict';
+
+/**
+ * Implements test results scoring for Interop 2022 as described in the RFC:
```

Comment:
Can you update this comment to say that we're using the scoring as defined in the 2022 RFC, but this computes scores for laters years too. Also, can you describe the interop score in words?